drop-in/src/main/java/com/adyen/checkout/dropin/internal/ui/CardComponentDialogFragment.kt

```diff
@@ -35,9 +34,11 @@ internal class CardComponentDialogFragment : BaseComponentDialogFragment() {
         super.onViewCreated(view, savedInstanceState)
         Logger.d(TAG, "onViewCreated")
 
-        // try to get the name from the payment methods response
-        binding.header.text = dropInViewModel.getPaymentMethods()
-            .find { it.type == PaymentMethodTypes.SCHEME }?.name
+        binding.header.text = if (isStoredPayment) {
+            storedPaymentMethod.merchantCustomizableName
+        } else {
+            paymentMethod.merchantCustomizableName
+        }
```

Comment:
This part is aligned with design team. Figma will be updated accordingly.
https://youtrack.is.adyen.com/issue/COAND-804/Allow-merchants-to-override-the-payment-method-names-that-show-up-in-Drop-in#focus=Comments-102-905829.0-0

Reply:
So it is okay that the specific brand name is shown for stored cards instead of the the generic "Credit card"?

Reply:
Yes exactly. I discussed it with Leo and Arjen. They think it makes sense to show the stored card name and that it is more descriptive.
They also want to include the card name in the drop in list for stored payment methods, but that will be part of bigger design changes.

I also asked Arjen to include the stored card payment component in Figma, since it was not mapped yet. He will also look into making the stored card component UI more accessible (color wise).

Reply:
Perfect 👍